client/state/data-layer/wpcom/read/lists/feeds/new/index.js

```diff
@@ -45,12 +45,8 @@ registerHandlers( 'state/data-layer/wpcom/read/lists/feeds/new/index.js', {
 					} ),
 				];
 			},
-			onError: ( action, error ) => {
-				return errorNotice(
-					translate( 'Could not add feed to list: %(message)s', {
-						args: { message: error?.message },
-					} )
-				);
+			onError: () => {
+				return errorNotice( translate( 'Unable to add feed to list.' ) );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Unable to add feed' )` *ES Score: 6*